### PR TITLE
Cirrus: Remove multi-arch podman image builds

### PR DESCRIPTION
**Pull request by cevich**
These jobs have been failing since early August due to technical/scripting problems.  Disable/remove entirely since a fix is unlikely to be implemented anytime soon.

Ref: https://github.com/containers/podman/pull/19720 - I gave up trying to debug it.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by edsantiago** [comment]
LGTM. "Failing" and "technical/scripting problem" are not defined. Is any future maintainer likely to wonder (or care) what the failure symptom is, and/or why the problem is insurmountable?

**Comment by cevich** [comment]
[Test of `[CI:BUILD]` looks okay.](https://cirrus-ci.com/build/5621839116894208)

**Comment by cevich** [comment]
> Is any future maintainer likely to wonder (or care) what the failure symptom is, and/or why the problem is insurmountable?

Hmmm, good point :thinking: The problem is mostly documented in https://github.com/containers/podman/pull/19720  I spent weeks trying to debug it without much luck.  More than likely I'll be the one worrying about it, and/or dropping my script entirely for a future implementation.   I'll drop mention of it in the PR description on the off-chance it helps someone...

**Comment by cevich** [comment]
Force-push: included abandoned jq-fix test pr. ref.

**Review by vrothberg — COMMENTED** [review]
/lgtm
/hold

Restarted the what I think was a flake

**Review by edsantiago — APPROVED** [review]

**Pull request merged**
